llm/llama-2/README.md

```diff
@@ -55,9 +55,11 @@ sky launch -c llama-serve -s chatbot-hf.yaml
 
 3. **Optional**: Try other GPUs:
 ```bash
-sky launch -c llama-serve-v100 -s chatbot-hf.yaml --gpus V100:8
+sky launch -c llama-serve-l4 -s chatbot-hf.yaml --gpus L4
 ```
 
+Here L4 is a GPU that is cost-effective for inference. You can find more details about the GPU types [here](https://cloud.google.com/blog/products/compute/introducing-g2-vms-with-nvidia-l4-gpus).
```

Comment:
is the below description better?

L4 is the latest generation GPU built for large inference AI workloads. Find more details [here](https://cloud.google.com/blog/products/compute/introducing-g2-vms-with-nvidia-l4-gpus).

Reply:
Sounds good! Updated.